app/controllers/application_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Chitter < Sinatra::Base
+
+  set :views, Proc.new { File.join(root, "../views") }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)